src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -52,10 +53,24 @@ public class Startup
 
```

Comment:
Are the SERVICE_NAME and Version used for anything now that the new config is being used?

Reply:
Nope, but I did forget to add them to the appsettings.